simplewebrtc.js

```diff
@@ -346,7 +348,7 @@ SimpleWebRTC.prototype.joinRoom = function (name, cb) {
                             }
                         });
                         self.emit('createdPeer', peer);
```

Comment:
Should this line include the roomName? Thus it should be: 
`self.emit('createdPeer', peer, roomName); `